File: 7KYU/Changing letters/solution.rb
def isVowel(c)
  c == 'a' or c == 'e' or c == 'i' or c == 'o' || c == 'u'  
end

def swap(st)
  st.chars.map {|c| isVowel(c) ? c.upcase : c}.join("")
end

File: 7KYU/Changing letters/solution.cs
using System.Linq;

public static class Kata
{
  private static bool IsVowel(char c)
     => c == 'a' || c=='e' || c=='i' || c == 'o' || c=='u';
  
  public static string Swap(string s)
    => s.Aggregate("", (s, c) => s += IsVowel(c) ? char.ToUpper(c) : c);
}